x-pack/plugins/enterprise_search/server/lib/connectors/delete_connector.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IScopedClusterClient } from '@kbn/core-elasticsearch-server';
+
+import { DeleteConnectorResponse } from '../../../common/types/connectors';
+
+export const deleteConnector = async (client: IScopedClusterClient, connectorId: string) => {
```

Comment:
We already have a delete_connector_by_id function (see this PR: https://github.com/elastic/kibana/pull/175105)

Reply:
huh I missed it, I will have to double check with Jedr what he has in his plate